CMakeLists.txt

```diff
@@ -194,8 +194,8 @@ install(FILES libtcmu.h libtcmu_common.h tcmu-runner.h
 install(FILES org.kernel.TCMUService1.service
   DESTINATION /usr/share/dbus-1/system-services)
 install(FILES tcmu-runner.conf DESTINATION /etc/dbus-1/system.d)
-if (SUPPORT_SYSTEMD)
```

Comment:
this is because you can invoke cmake as `cmake -DSUPPORT_SYSTEMD-ON` if you want systemd support. I'm not sure we can assume this from whether systemd is installed or not.